src/Aquifer.API/Endpoints/Resources/Content/CreateTranslation/Endpoint.cs

```diff
@@ -18,6 +18,18 @@ public override void Configure()
 
     public override async Task HandleAsync(Request request, CancellationToken ct)
     {
+        var user = await userService.GetUserFromJwtAsync(ct);
+        var isCommunityUser = userService.HasPermission(PermissionName.CreateCommunityContent);
```

Comment:
I think the absence of a permission would be better to determine this. Reason being that there could be situations where a user has extra permissions (say an admin type of user) and then they go into the wrong path. So I would check that they _don't_ have the `PermissionName.CreateContent` permission.

Reply:
is there a situation when a user would not have the `CreateContent` permission but also should not have the ability to `CreateCommunityContent` ?

Reply:
Because of the `Permissions(PermissionName.CreateContent, PermissionName.CreateCommunityContent);` in the configure you have to have at least one of them.